stock_picking_package_preparation_line/models/ir_config.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Francesco Apruzzese - Apulia Software srl
+# Copyright 2015-2018 Lorenzo Battistini - Agile Business Group
+# Copyright 2016 Alessio Gerace - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _inherit = 'res.config.settings'
+
+    default_picking_type_for_package_preparation_id = fields.Many2one(
+        'stock.picking.type',
+        related='company_id.default_picking_type_for_package_preparation_id')
+
+    @api.model
+    def default_get(self, fields):
+        res = super(ResConfigSettings, self).default_get(fields)
+        if res:
+            company = self.env.user.company_id
+            res['default_picking_type_for_package_preparation_id'] = \
```

Comment:
@SimoRubi Is this necessary as field is related ?

Reply:
Tanks for your review!
I have removed the entire method in https://github.com/OCA/stock-logistics-workflow/pull/447/commits/8ed7204eeedc97f7c2cb123f655664a1443e728e